From: Robert Luberda <robert@debian.org>
Date: Sun, 3 Jun 2018 16:28:38 +0200
Subject: Fix FTBFS with gcc-8

Increase buffer size to fix build failures like the one below:
 ../log.c:328:40: error: '%s' directive output may be truncated writing up to 4095 bytes into a region of size between 3838 and 4093 [-Werror=format-truncation=]
   snprintf (buf, sizeof (buf), "%s(%s) %s\n",

Bugs-Debian: https://bugs.debian.org/897789
---
 maa/log.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- maa/log.c
+++ maa/log.c
@@ -297,7 +297,7 @@ static void _log_base_va(
 {
    va_list ap_copy;
    time_t t;
-   static char   buf [4096] = "";
+   static char   buf [8192] = "";
    static char   buf_main [4096] = "";
    static char   buf_preamble [256] = "";
 
